### PR TITLE
Jaylyn C. Umana profile update

### PR DESCRIPTION
**Pull request by jc-umana**
Only change lines 1537 to 1565.

### CODE REVIEW

File: db/people.yml [hunk 1]
```diff
@@ -1548,9 +1548,10 @@ jumana:
   synthesis as well as first-principles calculations via density functional theory (DFT).
   education:
       - begin_year: 2019
```
Comment (sbillinge) [inline db/people.yml:1550]: indentation looks not right here.

**Comment by sbillinge** [comment]
@jc-umana I think we still need to work to merge this PR, do we?

File: db/people.yml [hunk 2]
```diff
@@ -1541,16 +1541,13 @@ jumana:
...
-  bio: Jaylyn C. Umana grew up on Long Island, NY and is currently an undergraduate student 
-  and researcher at Columbia University pursuing a degrees in both physics and mathematics. His primary research 
-  interests are in condensed matter with a focus on transition metal oxides. He has experience with
-  synthesis as well as first-principles calculations via density functional theory (DFT).
-  education:
-      - begin_year: 2019
-      end_year: 2023
-      degree: BA
-      institution: columbiau
+  avatar: https://github.com/\x16jc-umana.png
```
Comment (sbillinge) [inline db/people.yml:1544]: there is a typo in the name I think

Comment (jc-umana) [reply db/people.yml:1544]: Is it because the name field doesn't match any item in the aka field?

Comment (sbillinge) [reply db/people.yml:1544]: there is a rogue ``\x16`` that crept in somehow.  Just delete it.

**Comment by sbillinge** [comment]
Thanks, @jc-umana 

Wait a few minutes after I merge and look for the updates on billingegroup.github.io

**Pull request merged**